db/app.js

```diff
@@ -1,11 +1,26 @@
 const express = require("express");
 const fs = require("fs");
 const endpointsJson = require("../endpoints.json");
+const topics = require("../db/data/test-data/topics");
```

Comment:
Although this requiring in is fine for the endpoint.json it is not appropriate to get hold of the data for the response this way. We are making an API so we need to make sure the data is fetched from the database using a controller and a model. This is really important! Please do update this straight away and make sure you use the M(V)C way of working here on in